File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/inappbilling/GoogleInAppBillingActivity.java
package com.jdroid.android.sample.ui.google.inappbilling;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class GoogleInAppBillingActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return GoogleInAppBillingFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/inappbilling/GoogleInAppBillingActivity.kt
package com.jdroid.android.sample.ui.google.inappbilling

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class GoogleInAppBillingActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return GoogleInAppBillingFragment::class.java
    }
}
